sale_product_set/wizard/product_set_add.py

```diff
@@ -6,8 +6,8 @@
 
```

Comment:
Could you rename this file `sale_product_set_wizard.py` ?

Reply:
changes done, thanks!